aeron-driver/src/main/java/io/aeron/driver/DriverConductor.java

```diff
@@ -1916,6 +1916,11 @@ private void initLogMetadata(
     {
         final UnsafeBuffer logMetaData = rawLog.metaData();
 
+        System.out.println("SessionId: " + sessionId +
+            " streamId:" + streamId +
+            " initialTermId:" + initialTermId +
+            " mtuLength:" + mtuLength);
+
```

Comment:
What is this?

Reply:
Debug code for Aeron insights. I already removed it.